pkgs/applications/editors/neovim/default.nix

```
@@ -65,6 +73,18 @@ in
       cmake
       gettext
       pkgconfig
+    ]
+    ++ optionals devMode [
+      include-what-you-use  # for scripts/check-includes.py
+      jq                    # jq for scripts/vim-patch.sh -r
+      doxygen
+    ];
```

I'd do:

```suggestion
      # To generate dev docs
      doxygen
      jq                    # jq for scripts/vim-patch.sh -r
    ]
    ++ optionals doCheck [
      include-what-you-use  # for scripts/check-includes.py
    ];
```

I suggest this considering:

- I hope that adding doxygen doesn't mean dev docs are not built and installed unconditionally.
- jq doesn't bother anyone if it'll be there, since it'll most likely won't be referenced in the output.

---

all of these deps are just to help with development (in a `nix-shell -A neovim-unwrapped`), they dont change the build process.